apps/client/src/components/leaderboard/leaderboard.module.css

```diff
@@ -0,0 +1,16 @@
+.leaderboardTable {
+  display: flex;
+  height: 85%;
+  flex-direction: column;
+  padding: 0.5em;
+  gap: 2em;
+}
+.leaderBoardOne {
```

Comment:
Notice how here you named the class `leaderBoard` with capital  B and above you named it with lower case b.
Consistency is imortant!